src/aiperf/dataset/memory_map_utils.py

```
@@ -34,39 +38,69 @@
+    MemoryMapClientMetadata,
+)
 
 _logger = AIPerfLogger(__name__)
 
 
+def _import_zstandard():
+    """Lazy-import zstandard or raise a helpful error."""
+    try:
+        import zstandard
+
+        return zstandard
```

Ruff (0.15.2)

[warning] Consider moving this statement to an `else` block

(TRY300)